doc/level1.rst

```diff
@@ -672,6 +672,9 @@ attribute indicates the recipient address this header is valid for as
 usual, but may relate to any recipient in the ``To`` or ``Cc`` header.
 See example in :ref:`autocrypt-gossip-example`
 
+The ``Autocrypt-Gossip`` header can also be used
+to include keys for the address specified in the ``Reply-To`` header.
```

Comment:
It's pedantic, but we might want to mention that this is only useful in cases where that address differs from the ones in From/To/Cc